app/main/views/buyers.py

```diff
@@ -35,6 +37,22 @@
 
 @main.route('')
 def buyer_dashboard():
+    if flask_featureflags.is_active('DIRECT_AWARD_PROJECTS'):
+        user_briefs_total = len(data_api_client.find_briefs(current_user.id).get('briefs', []))
+        user_projects_total = len(data_api_client.find_direct_award_projects(current_user.id).get('projects', []))
+
+        return render_template(
+            'buyers/index.html',
+            user_briefs_total=user_briefs_total,
+            user_projects_total=user_projects_total
+        )
+
+    else:
+        return buyer_dos_requirements()
```

Comment:
Why do we want 2 end points showing exactly the same content? This is not a pattern we follow elsewhere.
I'd suggest a redirect and use the original url unless there's a really good reason not to.

```
return redirect(
    url_for(".buyer_dos_requirements",)
)
```

Reply:
We considered it didn't make sense to change the URLs, not to change the breadcrumbs. So when the new feature goes live, the URLs change and the breadcrumbs change at the same time (and no redirect is required, because both URLs still exist).

Reply:
> So when the new feature goes live, the URLs change

Do you mean the views?
Confused by this 😕

Reply:
Well they kinda both change: `/buyers/` effectively switches from being a DOS-specific view to being a general one, and the existing DOS-specific view moves to the new URL. We don't want any of this to show up until the feature flag is turned on.

Reply:
That means we're going to need to check every link to `'/buyers'` and `url_for('buyer_dashboard')` in the app to make sure they still make sense in context right?

Reply:
Right :) https://trello.com/c/mF0NJvQD/717-find-links-to-buyer-account-page-check-dos-impact

Reply:
See also https://github.com/alphagov/digitalmarketplace-briefs-frontend/pull/30

Reply:
Cool, so until the feature flag is on nothing changes. 
(`'/buyers'`, `buyer_dashboard `) point to where it always have
(`'/requirements/digital-outcomes-and-specialists'`, `buyer_dos_requirements `) points to the buyer dash too.

When the feature flag is on the new dash is up.

Reply:
This would have been better achieved by registering the urls based on the feature flag. Then we would have had no duplicate page and could have avoided the if statement that skips a whole view:
```

def buyer_dashboard():
    user_briefs_total = len(data_api_client.find_briefs(current_user.id).get('briefs', []))
    user_projects_total = len(data_api_client.find_direct_award_projects(current_user.id).get('projects', []))

    return render_template(
        'buyers/index.html',
        user_briefs_total=user_briefs_total,
        user_projects_total=user_projects_total
    )


def buyer_dos_requirements():
    user_briefs = data_api_client.find_briefs(current_user.id).get('briefs', [])

    draft_briefs = sorted(
        add_unanswered_counts_to_briefs([brief for brief in user_briefs if brief['status'] == 'draft'], content_loader),
        key=lambda i: datetime.strptime(i['createdAt'], DATETIME_FORMAT),
        reverse=True
    )
    live_briefs = sorted(
        [brief for brief in user_briefs if brief['status'] == 'live'],
        key=lambda i: datetime.strptime(i['publishedAt'], DATETIME_FORMAT),
        reverse=True
    )
    closed_briefs = sorted(
        [brief for brief in user_briefs if brief['status'] in CLOSED_BRIEF_STATUSES],
        key=lambda i: datetime.strptime(i['applicationsClosedAt'], DATETIME_FORMAT),
        reverse=True
    )

    breadcrumbs = [
        {
            "link": "/",
            "label": "Digital Marketplace"
        }
    ]

    if flask_featureflags.is_active('DIRECT_AWARD_PROJECTS'):
        breadcrumbs += [
            {
                "link": url_for("buyers.buyer_dashboard"),
                "label": "Your account"
            }
        ]

    return render_template(
        'buyers/dashboard.html',
        draft_briefs=draft_briefs,
        live_briefs=live_briefs,
        closed_briefs=closed_briefs,
        breadcrumbs=breadcrumbs,
    )


# Set routes depending on whether direct awards feature is active.
if flask_featureflags.is_active('DIRECT_AWARD_PROJECTS'):
    main.route('', endpoint='buyer_dashboard')(buyer_dos_requirements)
    main.route(
        '/requirements/digital-outcomes-and-specialists',
        endpoint='buyer_dos_requirements'
    )(buyer_dos_requirements)
else:
    main.route('', endpoint='buyer_dashboard')(buyer_dos_requirements)
    main.route('', endpoint='buyer_dos_requirements')(buyer_dos_requirements)

```